NitroxClient/GameLogic/ExosuitModuleEvent.cs

```diff
@@ -0,0 +1,47 @@
+﻿using NitroxClient.Communication.Abstract;
+using NitroxClient.GameLogic.Helper;
+using NitroxModel.DataStructures.GameLogic;
+using NitroxModel.DataStructures.Util;
+using NitroxModel.Helper;
+using NitroxModel.Logger;
+using NitroxModel.Packets;
+using UnityEngine;
+
+namespace NitroxClient.GameLogic
+{
+    public class ExosuitModuleEvent
+    {
+        private readonly IPacketSender packetSender;
+        private readonly IMultiplayerSession multiplayerSession;
+        private readonly Vehicles vehicles;
+
+        public ExosuitModuleEvent(IPacketSender packetSender, IMultiplayerSession multiplayerSession, Vehicles vehicles)
+        {
+            this.packetSender = packetSender;
+            this.multiplayerSession = multiplayerSession;
+            this.vehicles = vehicles;
+        }
+
+        public void BroadcastSpawnedArm(Exosuit exo)
+        {
+            string Guid = GuidHelper.GetGuid(exo.gameObject);
+            if (!string.IsNullOrEmpty(Guid))
```

Comment:
For future reference, this can never be null.  If a guid is not present, we set one to the object.